pkgs/build-support/go/gomod2nix/install/install.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"fmt"
+	"go/parser"
+	"go/token"
+	"io"
+	"os"
+	"os/exec"
+	"strconv"
+)
+
+const filename = "tools.go"
```

Comment:
Is it actually enforced that the file is called `tools.go` or just that the build tag is `tools`?

https://marcofranssen.nl/manage-go-tools-via-go-modules

`// +build tools`

I can't really find any docs on this